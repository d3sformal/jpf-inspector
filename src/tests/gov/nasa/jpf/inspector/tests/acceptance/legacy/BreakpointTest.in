create breakpoint pos=*:9
show breakpoint
run
# Loop: 5 times
 # Iteration
 wait
 run
 # Iteration
 wait
 run
 # Iteration
 wait
 run
 # Iteration
 wait
 run
 # Iteration
 wait
 run

# Termination:
wait
show breakpoint
# This should fail (because we already terminated):
print #thread