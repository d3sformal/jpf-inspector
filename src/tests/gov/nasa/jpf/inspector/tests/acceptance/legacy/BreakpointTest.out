1 :  state=ENABLED hits=0 hitsTotal=0 position=*:9
INFO: JPF created and connected, SuT is started
INFO: Breakpoint hit: [1 :  state=ENABLED hits=1 hitsTotal=1 position=*:9]
	SuT will now execute "return  gov.nasa.jpf.inspector.tests.acceptance.legacy.AssertSimpleTest.main([Ljava/lang/String;)V" at position *:9.
INFO: Breakpoint hit: [1 :  state=ENABLED hits=1 hitsTotal=2 position=*:9]
	SuT will now execute "return  gov.nasa.jpf.inspector.tests.acceptance.legacy.AssertSimpleTest.main([Ljava/lang/String;)V" at position *:9.
INFO: Breakpoint hit: [1 :  state=ENABLED hits=1 hitsTotal=3 position=*:9]
	SuT will now execute "return  gov.nasa.jpf.inspector.tests.acceptance.legacy.AssertSimpleTest.main([Ljava/lang/String;)V" at position *:9.
INFO: Breakpoint hit: [1 :  state=ENABLED hits=1 hitsTotal=4 position=*:9]
	SuT will now execute "return  gov.nasa.jpf.inspector.tests.acceptance.legacy.AssertSimpleTest.main([Ljava/lang/String;)V" at position *:9.
INFO: Breakpoint hit: [1 :  state=ENABLED hits=1 hitsTotal=5 position=*:9]
	SuT will now execute "return  gov.nasa.jpf.inspector.tests.acceptance.legacy.AssertSimpleTest.main([Ljava/lang/String;)V" at position *:9.
INFO: JPF Terminating
1 :  state=ENABLED hits=0 hitsTotal=5 position=*:9
ERR: This command may only be used when JPF has started and is paused. If JPF is not already started, then start it using 'run' and if it is already running, then pause it using 'break'. Then you will be able to execute this command. Alternatively, you may disable safe mode in the configuration file by setting 'jpf-inspector.safe-mode = false'.